src/lammpsio/data.py

```diff
@@ -39,6 +39,14 @@ class DataFile:
     unknown_bodies : list
         Data file body sections that will be ignored.
 
+
```

Comment:
```python
class DataFile:
    """LAMMPS data file."""
```

Lets add some description from the `README.md` file giving a general overview about the data file and specifically mentioning that not all things that could possibly go in a data file are supported.